realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -594,6 +594,22 @@ public double average(String fieldName) {
         return where().distinctAsync(fieldName);
     }
 
+    /**
+     * Returns a distinct set of objects from a specific class that fulfill the multiple fields
+     * condition. When multiple distinct fields are given, all unique combinations of tuples in such
+     * fields will be returned. If the result is sorted, the first object of a column will be returned
```

Comment:
Don't use column in the public API. Perhaps `In case of multiple matches, it is undefined which object is returned, unless the result is sorted, then the first object will be returned.`